data/src/main/java/com/globant/equattrocchio/data/response/Result.java

```diff
@@ -8,14 +8,14 @@
 public class Result {

```

Comment:
Rename by more accord name. For example

GetImagesResponse

Reply:
done